versions/latest/modules/en/nav.adoc

```diff
@@ -1,5 +1,7 @@
 * xref:introduction.adoc[Introduction]
 
+* xref:k3k.adoc[K3k: Kubernetes in Kubernetes]
```

Comment:
The `K3k` in the label is capitalized here and differs from the actual page's title where it's lowercase. Is that intended?